Add `economyFee` field in /api/fees/recommended API

nymkappa opened the pull request:
Redo of https://github.com/mempool/mempool/pull/1734
Context https://github.com/mempool/mempool/issues/1726

Added `economyFee` field in `/api/fees/recommended` API which is always `2 x minimumFee`.

hunicus reviewed (approved):
Nitpick but I expected to see economyFee appear before minimumFee to keep the order of highest-to-lowest priority.

nymkappa commented:
> Nitpick but I expected to see economyFee appear before minimumFee to keep the order of highest-to-lowest priority.

Valid Nitpick! Fixed it, rebased on master and force pushed.

wiz reviewed (approved):
Tested ACK @ v2.4.0-dev [[1cb772da](https://github.com/mempool/mempool/commit/1cb772da)]
